builtin/credential/aws/backend_test.go

```diff
@@ -477,13 +477,13 @@ func TestBackend_ConfigClient(t *testing.T) {
 
 	stepCreate := logicaltest.TestStep{
 		Operation: logical.CreateOperation,
-		Path:      "config/client",
+		Path:      "config/core.Client",
```

Comment:
Why have all of these been changed to core.Client?

Reply:
oops, typo.  fixed